getting-started/bun.md

```diff
@@ -23,6 +24,13 @@ cd my-app
 bun install
 ```
 
+### 2.2. Setup an existing project
+On an existing Bun project, we only need to install `hono` dependencies on the project root directory via
+
+```sh
+bun install hono
```

Comment:
I think [`bun add` ](https://bun.sh/guides/install/add) is better. What do you think of it?

Reply:
Sure, I am going to change it.